pageobjects/item.page.js

```diff
@@ -84,6 +92,14 @@ class ItemPage extends PageMixture {
 		return reference.$( this.constructor.ITEM_WIDGET_SELECTORES.PROPERTY_INPUT );
 	}
 
+	selectSuggestedProperty( propertyId ) {
```

Comment:
Not sure if calling it `selectSuggestedProperty` is ideal if it can also suggest other entities. Wouldn't `selectSuggestedEntity` not be more accurate?

Reply:
good idea .. I did actually use it to select entities in lexeme.. gotta update both :+1: